components/ai_chat/resources/page/components/welcome_guide/index.tsx

```diff
@@ -5,55 +5,138 @@
+    title: getLocale(S.CHAT_UI_WELCOME_CARD_TRULY_PRIVATE_TITLE),
+    description: getLocale(S.CHAT_UI_WELCOME_CARD_TRULY_PRIVATE_SUBTITLE),
+    illustration: TrulyPrivateIllustration,
+  },
+  {
+    title: getLocale(S.CHAT_UI_WELCOME_CARD_TABS_AND_FILES_TITLE),
+    description: getLocale(S.CHAT_UI_WELCOME_CARD_TABS_AND_FILES_SUBTITLE),
+    illustration: ChatWithIllustration,
+  },
+  {
+    title: getLocale(S.CHAT_UI_WELCOME_CARD_AUTOMATE_WORK_TITLE),
+    description: getLocale(S.CHAT_UI_WELCOME_CARD_AUTOMATE_WORK_SUBTITLE),
+    illustration: AutomateIllustration,
+  },
+]
+
 function WelcomeGuide() {
-  const conversationContext = useConversation()
+  const aiChatContext = useAIChat()
+
+  // State
+  const [visibleCard, setVisibleCard] = React.useState(0)
+  const [userSelectedCard, setUserSelectedCard] = React.useState<
+    number | undefined
+  >(undefined)
+
+  // Computed
+  const subtitle = formatLocale(S.CHAT_UI_WELCOME_GUIDE_SUBTITLE, {
+    $1: (content) => <span className={styles.subtitlePurple}>{content}</span>,
+  })
+
+  // Methods
+  const handleLearnMoreClicked = React.useCallback(() => {
+    const mojomUrl = new Url()
+    mojomUrl.url = 'https://brave.com/leo/'
 
-  const summarizeNow = () => {
-    conversationContext.conversationHandler?.submitSummarizationRequest()
-  }
+    aiChatContext.uiHandler?.openURL(mojomUrl)
+  }, [aiChatContext.uiHandler])
+
+  // Effects
+  React.useEffect(() => {
+    // Only start auto-cycling if user hasn't selected a card
+    if (userSelectedCard !== undefined) {
+      return
+    }
+
+    const interval = setInterval(() => {
+      setVisibleCard((prevIndex) => (prevIndex + 1) % welcomeCards.length)
+    }, 10000) // 10 seconds
+
+    return () => clearInterval(interval)
+  }, [userSelectedCard])
 
   return (
-    <div className={styles.box}>
+    <div className={styles.wrapper}>
       <div className={styles.header}>
-        <h1 className={styles.title}>
+        <span className={styles.title}>
           {getLocale(S.CHAT_UI_WELCOME_GUIDE_TITLE)}
-        </h1>
-        <h2 className={styles.subtitle}>
-          {getLocale(S.CHAT_UI_WELCOME_GUIDE_SUBTITLE)}
-        </h2>
+        </span>
+        <span className={styles.subtitle}>{subtitle}</span>
       </div>
-      <div className={`${styles.card} ${styles.siteHelpCard}`}>
-        <h4 className={styles.cardTitle}>
-          {getLocale(S.CHAT_UI_WELCOME_GUIDE_SITE_HELP_CARD_TITLE)}
-        </h4>
-        {conversationContext.associatedContentInfo.length > 0 ? (
-          <>
-            <p>
-              {getLocale(S.CHAT_UI_WELCOME_GUIDE_SITE_HELP_CARD_WITH_ACTION)}
-            </p>
-            <div className={styles.actions}>
-              <Button
-                kind='outline'
-                onClick={summarizeNow}
+      <div className={styles.cardWrapper}>
+        {welcomeCards.map((card, i) => {
+          const isSelected =
+            userSelectedCard !== undefined
+              ? userSelectedCard === i
```

Comment:
mega nit for consistency:
```suggestion
              ? i === userSelectedCard
```